server/db/mongodb/adapter.go

```diff
@@ -1264,14 +1266,16 @@ func (a *adapter) TopicsForUser(uid t.Uid, keepDeleted bool, opts *t.QueryOpt) (
 			} else {
 				usrq = append(usrq, uid1.String())
 			}
-			topq = append(topq, sub.Topic)
+			topq = append(topq, tname)
 
 			// grp subscription
 		} else {
-			topq = append(topq, sub.Topic)
+			// Convert channel names to topic names.
+			tname = t.GrpFromChn(tname)
```

Comment:
ChnToGrp?

Reply:
Done. I renamed it but missed in a couple of places.